dev/sparktestsupport/modules.py

```diff
@@ -769,7 +769,7 @@ def __hash__(self):
 
 docker_integration_tests = Module(
     name="docker-integration-tests",
-    dependencies=[],
+    dependencies=[sql],
```

Comment:
hm, the problem is that the tests will still be skipped because `ENABLE_DOCKER_INTEGRATION_TESTS` is set only when explicitly there are some changes in `docker_integration_tests` module alone. (e.g., when `sql` module is picked alone, `ENABLE_DOCKER_INTEGRATION_TESTS` is not set, and the related tests are skipped).

This logic was followed from the Kinesis one but maybe we should just remove `ENABLE_DOCKER_INTEGRATION_TESTS` away and treat it like other test cases.

This env approach was used because of the concern of flakiness IIRC but actually the test seems pretty stable (?).

Reply:
alternatively we can copy:

```
    environ=None if "GITHUB_ACTIONS" not in os.environ else {
        "ENABLE_DOCKER_INTEGRATION_TESTS": "1"
    },
```

to `sql` module too to be extra conservative.

Reply:
Got it. In that case, shall we move that part from here to `sql` module?

Reply:
Then, it will happen at `catalyst` module change, too. I'll put it to both `catalyst` and `sql` module.

Reply:
Yeah, that should be fine for now 👍